### PR TITLE
Fix CheckStyle issues and enable CheckStyle in test code.

### PR DESCRIPTION
**Pull request by chivesrs**
Add newlines before curly braces (LeftCurlyCheck).
Remove unused imports (UnusedImportsCheck).
Add whitespaces as needed (WhitespaceAfterCheck).

Not much else to say about this PR, just a bunch of cleanup.
The issues regarding asserts and WhiteSpaceAfterCheck are tests that should be using JUnit asserts and not Java assert keyword. That will be fixed in #8081.

### CODE REVIEW

**Comment by Adam-** [comment]
CI runs checkstyle, and it is passing, so how is this fixing anything?

**Comment by Adam-** [comment]
Apparently CI is not running checkstyle on tests, if you want to enable that: https://maven.apache.org/plugins/maven-checkstyle-plugin/faq.html#How_do_I_include_the_test_directory_in_Checkstyle then it would make sense to go through and fix those errors.

**Comment by chivesrs** [comment]
Added flag to Maven's config to enable CheckStyle on test code.

**Comment by chivesrs** [comment]
It looks like I need to either:
1. Merge #8081 into this PR
2. Enable the flag in its own PR after this and #8081 is merged

What's your opinion?

**Comment by Adam-** [comment]
We can merge 8081 independently of this since it is wrong regardless of checkstyle.

**Comment by chivesrs** [comment]
Understood. I'll add a bunch of spaces to fix the whitespace checkstyle errors in this PR then.

File: runelite-client/src/test/java/net/runelite/client/plugins/hiscore/HiscorePanelTest.java [hunk 1]
```diff
@@ -31,7 +31,9 @@
 	@Test
 	public void testConstructor()
 	{
-		new HiscorePanel(new HiscoreConfig() {});
+		new HiscorePanel(new HiscoreConfig()
+		{
+
```
Comment (Nightfirecat) [inline runelite-client/src/test/java/net/runelite/client/plugins/hiscore/HiscorePanelTest.java]: This empty line can be removed.

Comment (chivesrs) [reply runelite-client/src/test/java/net/runelite/client/plugins/hiscore/HiscorePanelTest.java]: Sure, removed. I wasn't aware of the style on this particular scenario.

**Pull request merged**